src/openproduct/producttypen/tests/test_validators.py

```diff
@@ -0,0 +1,34 @@
+from django.core.exceptions import ValidationError
```

Comment:
These tests can be moved to  `tests/test_producttype.py`. all model validation tests are located in `tests/<model>.py`